### PR TITLE
add domesticShipping and internationalShipping to artwork

### PR DESCRIPTION
**Pull request by oxaudo**
Not sure if we have to account for some other rules here but for now just putting the default implementation of converting cents to money string. Also seems like in most cases where we use this - the cents are ignored in the full amount. So doing it here as well, not sure that is correct. 

Some formatting changes just from my re save of the file...

### CODE REVIEW

File: src/schema/artwork/index.js [hunk 1]
```diff
@@ -570,20 +518,20 @@ export const artworkFields = () => {
         return partnerLoader(partner.id).catch(() => null)
       },
     },
-    price: {
-      type: GraphQLString,
-    },
+    price: { type: GraphQLString },
+    domesticShipping: amount(
```
Comment (oxaudo) [inline src/schema/artwork/index.js]: Also was not sure if I'm supposed to use underscore_case or capitalCase for the name here. What are the current rules here?

Comment (mzikherman) [reply src/schema/artwork/index.js]: I think we're settled on 🐫 for newer things!

Comment (mzikherman) [reply src/schema/artwork/index.js]: I'm assuming the currency here is pretty much implied/assumed to all be USD?

As an FYI (now that I think about it I should extract this into the `money` schema), here's how we result in a displayable string for artist auction results: https://github.com/artsy/metaphysics/blob/1a055771148a142845d85226c9758c291898d579/src/schema/auction_result.js#L199-L217

It handles currency and the cents, and thus properly handles yen (not dividing cents by 100), correctly including the GBP symbol, etc.

But for now I think just using the `amount` helper should totally work!

File: src/schema/artwork/index.js [hunk 2]
```diff
@@ -570,20 +518,20 @@ export const artworkFields = () => {
...
+      ({ domestic_shipping_fee_cents }) => domestic_shipping_fee_cents
```
Comment (starsirius) [inline src/schema/artwork/index.js]: We chatted this on [slack](https://artsy.slack.com/archives/C9ZJYFDNF/p1533108372000187), but I am still not sure if MP is able to get these fees from Gravity, given it's in [`:all`](https://github.com/artsy/gravity/blob/637a58e0172a6ce5dc2d5ac36eedd2aa18ce6578/app/models/domain/artwork.rb#L1013).

**Pull request merged**